samples/Vue/basic.vue

```diff
@@ -0,0 +1,21 @@
+<style>
+.red {
+  color: #f00;
+}
+</style>
+
+<template>
+<div>
+  <h2 v-class="red">{{msg}}</h2>
+</div>
+</template>
+
+<script>
+module.exports = {
```

Comment:
We set [HTML to `markup`](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L1272) to I think we should set Vue to `markup` too ;)